install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -390,6 +390,15 @@ parameters:
 <1> Azure storage account SKU tier. Default is empty. *Note:* Premium VM can attach both _Standard_LRS_ and _Premium_LRS_ disks, Standard VM can only attach _Standard_LRS_ disks, Managed VM can only attach managed disks, and unmanaged VM can only attach unmanaged disks.
 <2> possible values are `shared` (default), `dedicated`, and `managed`. When `kind` is `shared`, all unmanaged disks are created in a few shared storage accounts in the same resource group as the cluster. When `kind` is `dedicated`, a new dedicated storage account gets created for the new unmanaged disk in the same resource group as the cluster. When `kind` is `managed`, a new managed disk gets created.
 
+[IMPORTANT]
+====
+Azure StorageClass is revised in {product-title} version 3.7. If you have upgraded from a previous version, either:
+
+ * specify the property `kind: dedicated` to continue using the Azure StorageClass created before the upgrade.
```

Comment:
* To continue using the Azure StorageClass created before the upgrade, specify the property `kind: dedicated`.